src/components/look/search-box/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+import { Input } from 'antd';
+import styled from 'styled-components';
+
+import { P } from '@src/component/atoms';
+
+export type SearchBoxProps = {
+  onSearch: (keyword: string) => void;
+};
+
+function SearchBox({ onSearch }: SearchBoxProps) {
+  const [keyword, setKeyword] = useState('');
+
+  const handleChange = (e) => {
+    const {
+      target: { value },
+    } = e;
+    setKeyword(value);
```

Comment:
```suggestion
    setKeyword(e.target.value);
```

아래에서 e.target.value라고 썼으니 통일해주는게 좋을 것 같아요!

Reply:
10d728c